src/cmd/server.ts

```diff
@@ -133,17 +139,41 @@ async function main() {
     validate: false,
   });
 
+  const app = express();
+
+  const limiter = rateLimit({
+    windowMs: 15 * 60 * 1000, // 15 minutes
+    max: 100, // Limit each IP to 100 requests per `window` (here, per 15 minutes)
```

Comment:
We should make this controllable by env vars, so we can easily update the parameters in production

Reply:
Makes sense. Will make that change

Reply:
Done